instagram/src/components/Authentication/Authenticate.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import "./Auth.css";
+import LoginPage from "./LoginPage";
+import SearchBar from "../SearchBar/SearchBar";
+import PostsPage from "../PostContainer/PostsPage";
+
+class Auth extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      loggedIn: false,
+      hello: "Hello",
+      filteredPosts: [],
+      filterTarget: "",
```

Comment:
its not wrong but for better name convention, name it what it is. 
An ideal name would be searchInput since that is what it is. So when other ppl read your code it is more straightforward to what its for & whats it doing .